### DISCUSSION TITLE
Is it possible to have different active projects for different windows?

### DISCUSSION THREAD

**Discussion by rhyek** [dbeaver/dbeaver#35342] [Q&A]
Sometimes I'm working on different projects throughout the week and would like to have one DBeaver instance with one project and all related queries to that project separate from another DBeaver instance with a different project.

Currently, when having two windows open, changing the Active Project in one changes it in the other, too.

**Comment by E1izabeth** [comment]
Currently it's not possible, unless you duplicate workspace and run two instances of DBeaver on different workspaces
